addon/components/o-s-s/chip.stories.js

```diff
@@ -0,0 +1,81 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+const SkinTypes = ['default', 'primary', 'success', 'danger'];
+
+export default {
+  title: 'Components/OSS::Chip',
+  component: 'chip',
+  argTypes: {
+    skin: {
+      description: 'Adjust appearance',
+      table: {
+        type: {
+          summary: SkinTypes.join('|')
+        },
+        defaultValue: { summary: 'primary' }
+      },
+      options: SkinTypes,
+      control: { type: 'select' }
+    },
+
+    label: {
+      type: { required: true },
+      description: 'Text content of the chip',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: {
+        type: 'text'
+      }
+    },
+
+    onRemove: {
+      type: { required: true },
+      description: 'A callback that sends the click event on cross to the parent component'
```

Comment:
`A callback triggered when the cross icon has been clicked.`

There are no parameters being sent.